src/azure-cli/azure/cli/command_modules/aro/_params.py

```diff
@@ -10,13 +10,15 @@
 from azure.cli.command_modules.aro._validators import validate_disk_encryption_set
 from azure.cli.command_modules.aro._validators import validate_domain
 from azure.cli.command_modules.aro._validators import validate_pull_secret
+from azure.cli.command_modules.aro._validators import validate_sdn
```

Comment:
I don't think SDN choice is in any GA API version, and is not planned to be a choice. I think everything related to SDN should be removed from this PR - I'll flag them.

Reply:
I think I got them all, thanks for the extra set of eyes.